docs/usage/configuration/config_documentation.md

```diff
@@ -1122,14 +1122,18 @@ Caching can be configured through the following sub-options:
 * `cache_autotuning` and its sub-options `max_cache_memory_usage`, `target_cache_memory_usage`, and
    `min_cache_ttl` work in conjunction with each other to maintain a balance between cache memory 
    usage and cache entry availability. You must be using [jemalloc](https://github.com/matrix-org/synapse#help-synapse-is-slow-and-eats-all-my-ramcpu) 
-   to utilize this option, and all three of the options must be specified for this feature to work.
+   to utilize this option, and all three of the options must be specified for this feature to work. This option
+   defaults to off, enable it by providing values for the sub-options listed below. Please note that the feature will not work
+   and may cause unstable behavior (such as excessive empyting of caches) if all of the values are not provided.
+   When providing values for memory usage use  M for MB and K for KB. 
```

Comment:
Is this line necessary after #12777?

Reply:
I think it makes sense from a user-friendly standpoint, just in case someone was linked to this option or hadn't looked over the full manual. I did change it to say KiB and MiB and linked the comment to the config convention section for completeness.

Reply:
Great point, I've updated it, let me know what you think.